spec/rubocop/cop/layout/redundant_line_break_spec.rb

```diff
@@ -172,6 +172,37 @@
         RUBY
       end
 
+      it 'registers an offense when using `&&` before a backslash newline' do
+        expect_offense(<<~RUBY)
+          foo && \\
+          ^^^^^^^^ Redundant line break detected.
+            bar
+        RUBY
+
+        expect_correction(<<~RUBY)
+          foo && bar
+        RUBY
+      end
+
+      it 'does not register an offense when using `&&` after a backslash newline' do
+        expect_no_offenses(<<~RUBY)
+          foo \\
+            && bar
+        RUBY
+      end
+
+      it 'registers an offense when using `||` before a backslash newline' do
```

Comment:
Seems to me you're missing the "after" test case here. (you have 2 for `&&` and just 1 for `||`)

Reply:
Oops! I've added the missing test.